app/helpers/tul_cdm_helper.rb

```diff
@@ -589,6 +589,35 @@ def get_all_facets(facet_field)
       :sort => response.params[:"f.#{facet_field}.facet.sort"] || response.params["facet.sort"])
   end
 
+  def collections_fields
```

Comment:
@skng5 I'm a little hesitant to hard code his, as it is liable to get out of sync when new collections are available. Is there something I'm missing about why we shouldn't just do a ` DigitalCollection.pluck(:name).unshift(["All Collections", ""])` ?